create-snowpack-app/app-template-11ty/_output/index.html

```diff
@@ -2,15 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="/static/favicon.png" />
+    <link rel="icon" href="%PUBLIC_URL%static/favicon.png" />
```

Comment:
Something that I realized in #1615: we may want to diverge from CRA here as a part of our v3.0 release: instead of requiring PUBLIC_URL, we can just auto-add it to any absolute imports/paths found here. We already need to do that across the rest of the codebase anyway (for things like resolving `import 'foo.svg'`, adding preload links, etc) so I actually don't think this is too much of a stretch from here.

In that case, no change would be needed here for GH pages in v3.0. To help with the migration, we could add a short blurb to snowpack.dev for the v2.x docs to talk about adding this yourself.

Reply:
Well if this particular pr as it stands does not get merged that is ok with me. There is more to do on a refocused path to gh pages. That path needs #1612 fixed immediately and attention to #1608 soon.

Reply:
sounds good, can take a look tomorrow